utility/meerkat_status.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/python
+import time
+import re
+import os
+from prettytable import PrettyTable
+from colorama import Fore
+import operator
+from katcorelib import standard_script_options, verify_and_connect
+
+
+def sync_hrs_remaining(ants, band):
+    """Returns the hours left for which the digitiser remains synchronised"""
+    sensor = "dig_{}_band_time_remaining".format(band)
+    for ant in ants:
+        try:
+            time_left = ant.sensor[sensor].get_value()
+            if time_left:
+                break
+        except:
```

Comment:
Use "except Exception" instead. Linters will typically complain if you just have an "except:" by itself.